server/runtime.go

```diff
@@ -73,9 +73,19 @@ func (s *Server) CreatePodSandbox(ctx context.Context, req *pb.CreatePodSandboxR
 		logDir = fmt.Sprintf("/var/log/ocid/pods/%s", name)
 	}
 
-	// TODO: construct /etc/resolv.conf based on dnsOpts.
-	dnsOpts := req.GetConfig().GetDnsOptions()
-	fmt.Println(dnsOpts)
+	dnsServers := req.GetConfig().GetDnsOptions().GetServers()
```

Comment:
I had the same concern in the other PR but apparently it never returns nil (even if the go to protobuf produces not nil safe methods)